src/lib/src/items-list.ts

```diff
@@ -16,13 +16,20 @@ export class ItemsList {
         this._multiple = multiple;
     }
 
+    get value(): NgOption | NgOption[] {
+        if (this._multiple) {
+            return this._selected;
+        }
+        return this._selected[0];
+    }
+
     select(item: NgOption) {
```

Comment:
Add separate method for unselecting item then in multiselect click on clear icon https://github.com/ng-select/ng-select/blob/master/src/lib/src/ng-select.component.html#L21

```
unSelect(item: ngOption) {}
```

and separate `toggleSelect(item: NgOption)` to use in dropdown when it is needed to toggle selected value.